src/search/SearchMessagesScreen.js

```diff
@@ -71,6 +71,13 @@ class SearchMessagesScreen extends PureComponent {
     const { actions, auth, subscriptions, twentyFourHourTime, flags } = this.props;
     const noResults = !!this.query && !isFetching && !messages.length;
 
+    if (noResults) {
```

Comment:
You should not be duplicating the Screen component.
I did split the SearchMessagesScreen in three so you can do that.
Add the same code, but in SearchMessagesCard now.